src/_guides/language/effective-dart/usage.md

```diff
@@ -19,21 +19,58 @@ but *maintainers* of it sure will.
 
 These guidelines help you compose your program out of multiple files in a
 consistent, maintainable way. To keep these guidelines brief, they use "import"
-to cover `import` and `export` directives. The guidelines apply equally to both.
+to cover `import` and `export` directives. The guidelines apply equally to both. 
+
+### DON'T use library directives unless attaching doc comments or annotations
+
+[Library-level doc comments][] and [annotations][]
+must prepend a `library` declaration at the start of a file.
```

Comment:
This reads a little strange to me because it's not clear that the library directive is a solution to a problem. Maybe:

> The only reason to use a library directive is to be able to attach library-level [doc comments] or [metadata annotations][].

?